src/main/java/step2/model/Ladder.java

```diff
@@ -0,0 +1,26 @@
+package step2.model;
+
+import step2.strategy.RandomDrawStrategy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Ladder {
+    private List<Line> ladder;
+
+    public Ladder(int height, int countOfPeople) {
+        ladder = Stream.generate(Line::new)
+                .limit(height)
+                .collect(Collectors.toList());
+        ladder.forEach(it -> it.drawNewLine(countOfPeople, new RandomDrawStrategy()));
+    }
```

Comment:
```suggestion
    private Ladder(List<Line> ladder) {
        this.ladder = ladder;
    }
    
    public Ladder of (List<Line> ladder) {
      return new Ladder(ladder);
    }

    public Ladder of (int height, int countOfPeople) {
        return Stream.generate(Line::new)
                     .limit(height)
                     .pick(it -> it.drawNewLine(countOfPeople, new RandomDrawStrategy()))
                     .collect(collectingAndThen(toList(), Ladder::of));
    }
```
이런식으로 생성자에게는 생성자 역할만 부여해주세요!
그리고 forEach 대신 pick을 이용하면 위와 같이 표현할 수 있습니다.

Reply:
height도 원시값 포장으로 만들어서 사용해주세요!